packages/i18n/I18nDecorator/I18nDecorator.js

```diff
@@ -143,12 +143,70 @@ const I18nDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	return I18nDecorator;
 });
 
+/**
+ * Default config for `I18nContextDecorator`.
+ *
+ * @memberof i18n/I18nDecorator.I18nContextDecorator
+ * @hocconfig
+ */
 const contextDefaultConfig = {
+	/**
+	 * The prop name for `locale` property of context.
```

Comment:
```suggestion
	 * The prop name for `locale` property of i18nContext.
```
Also, it would be better to update rtlProp and updateLocaleProp too.